tttx/pars/R3BTttxContFact.cxx

```diff
@@ -0,0 +1,77 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2024 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// ---------------------------------------------------------------
+// -----            R3BTttxContFact source file              -----
+// -----    Created 18/01/2024 by J.L. Rodriguez-Sanchez     -----
+//----------------------------------------------------------------
+
+#include <FairLogger.h>
+#include <FairRuntimeDb.h>
+
+#include "R3BTGeoPar.h"
+#include "R3BTttxContFact.h"
+#include "R3BTttxStripCalPar.h"
+// #include "R3BTttxMappingPar.h"
+
+#include <TClass.h>
+
+static R3BTttxContFact gR3BTttxContFact;
+
+R3BTttxContFact::R3BTttxContFact()
+{
+    // Constructor (called when the library is loaded)
+    fName = "R3BTttxContFact";
+    fTitle = "Factory for parameter containers in libR3BTttx";
+    setAllContainers();
+    FairRuntimeDb::instance()->addContFactory(this);
+}
+
+void R3BTttxContFact::setAllContainers()
+{
+    // Creates the Container objects with all accepted contexts and adds them to
+    // the list of containers for the TTTX library.
+
+    FairContainer* p1 = new FairContainer("tttxMappingPar", "TTTX Mapping Parameters", "TttxMappingParContext");
+    p1->addContext("TttxMappingParContext");
+    containers->Add(p1);
+
+    FairContainer* p2 = new FairContainer("tttxStripCalPar", "TTTX StripCal Parameters", "TttxCalParContext");
+    p2->addContext("TttxCalParContext");
+    containers->Add(p2);
```

Comment:
We don't have a mapping par, but would have cal2hit par. Could to define p1 as stripcalpar and p2 as hitpar?

Reply:
I am still working on it, so I will add the class for Cal2Hit

Reply:
Hello @ryotani 
Which parameters do you need to move data from Cal to Hit?

Reply:
@jose-luis-rs For HitPar I think parameters as
- NumDetPar (default = 2)
- NumZReconstructPar (default = 2)
- ZReconstructPar (TArrayF with fNumDet * fNumZRecPar elements)

Also for the CalPar, if these parameters can be added that's great.
- NumStrips
- StripsInUse (TArrayF with fNumDet * fNumStrips elements)

Reply:
I will do it today

Reply:
@ryotani, let me know if you are happy with this to merge the code. Thanks